lib/hanami/assets/bundler.rb

```diff
@@ -116,8 +128,11 @@ def generate_manifest
 
       # @since 0.1.0
       # @api private
-      def store_manifest(asset, target)
-        @manifest[_convert_to_url(::File.expand_path(asset))] = _convert_to_url(::File.expand_path(target))
+      def store_manifest(asset, target, sri_value)
+        @manifest[_convert_to_url(::File.expand_path(asset))] = {
+          target: _convert_to_url(::File.expand_path(target)),
```

Comment:
@accuser 
> I think that it would be a good idea to drop the target for the actual MD5 checksum, generating the asset digest on the fly:

That isn't practical, because at the runtime we should separate the base name from the extension to inject the checksum value. But the problem is to get the right separation (eg `/assets/jquery.min.js`). We already handle that, while compiling. We should duplicate that logic here to guarantee correctness..

Oh, and don't forget that this would be more time consuming, than just fetch `"target"` from the JSON.

My opinion is to leave it as it is :wink: